charts/astronomer/templates/commander/commander-grpc.yaml

```diff
@@ -0,0 +1,68 @@
+############################
+## Commander GRPC Ingress ##
+############################
+{{- if and .Values.global.dataplane.baseDomain .Values.global.dataplane.enabled }}
+kind: Ingress
+apiVersion: {{ template "apiVersion.Ingress" . }}
+metadata:
+  name: {{ .Release.Name }}-commander-grpc-ingress
```

Comment:
should we get rid of commander name just have grpc-ingress

Reply:
i'd say if we're going to get rid of any part of it, we should get rid of `grpc`. calling it that will be bad if we decide to use this ingress for additional protocols.